src/Step/StepContollers/StepSqrtController.php

```diff
@@ -5,16 +5,28 @@
 namespace Mathematicator\Step\Controller;
 
 
+use function count;
 use Mathematicator\Engine\Step;
 use Mathematicator\Step\StepFactory;
 use Nette\Utils\ArrayHash;
+use function strlen;
+use function substr;
 
 final class StepSqrtController implements IStepController
 {
 
 	/** @var Step[] */
 	private $steps = [];
 
+	/** @var StepFactory */
+	private $stepFactory;
```

Comment:
StepFactory is static in new version and this property is not used anywhere

Reply:
I'll fix it.